15special.tex

```diff
@@ -168,7 +168,7 @@ \subsection{Jitter}
 
 If the original initial value is at or near the middle of the min-max range, then for each 0.1 of jitter, the range of jitters extends about 0.25 sigmas to either side of the original value (though as the total jitter increases the relationship varies more than this), and the average absolute jitter is about half of that.  For values far from the middle of the min-max range, the resulting jitter is skewed in parameter space, and may hit the bound, invoking the resetting mentioned above. 
 
-To evaluate the jittering, the bounds, and the original initial values, a jitter\_info table is available from r4ss, including sigma, CV and InitLocation columns (the latter referring to location within the cumulative normal – too close to 0 or 1 indicates a potential issue).
+To evaluate the jittering, the bounds, and the original initial values, a jitter\_info table is available from r4ss, including sigma, CV and InitLocation columns (the latter referring to location within the cumulative normal - too close to 0 or 1 indicates a potential issue).
```

Comment:
Jim Hastie has slowly worn me down into believing in the importance of different dashes. I think that this one should be an m-dash, so should appear in latex like `cumulative normal---too close`. See https://site.uit.no/english/punctuation/hyphen/.

In all the other cases I see, the change you've made to the single `-` seems right.